config/quota-file.json

```diff
@@ -19,6 +19,7 @@
     "css:config/ldp/authorization/allow-all.json",
     "css:config/ldp/handler/default.json",
     "css:config/ldp/metadata-parser/default.json",
+    "css:config/ldp/metadata-parser/parsers/content-length.json",
```

Comment:
It is very important that all the default configurations have the same amount of imports in the same order, so it's clearer to users which features are enabled. So if extra lines would have to be added, it would have to be add the end. Look at `sparql-file-storage.json` for an example.

I'm also not sure if it makes more sense to add it in the default configs here. Another option is to add them in the `storage/backend/*-quota-file.json` configs to make sure it's always included when someone imports those specific features. Not sure tbh so other opinions are welcome.